1-JavaScript/Week4/homework/ex5-wallet/index.js

```diff
@@ -104,7 +104,7 @@ const quiz = {
       b: 'Transferring € 50,00 from Jack to undefined', 
       c: 'Transferring € 50,00 from Jack to Jane' 
     },
-    answer: undefined,
+    answer: a,
```

Comment:
@Mehran-Banbersta the answer should be a String 
```suggestion
    answer: 'a',
```